beacon-chain/powchain/service.go

```diff
@@ -388,6 +391,9 @@ func (s *Service) dialETH1Nodes() (*ethclient.Client, *gethRPC.Client, error) {
 	if err != nil {
 		return nil, nil, err
 	}
+	if s.auth != "" {
+		httpRPCClient.SetHeader("Authorization", s.auth)
+	}
```

Comment:
Is it possible to write an interactive test for this part, sending a request and verifying that the correct header is present on the client side? After all this is the core of the PR.